Dockerfile

```
@@ -0,0 +1,37 @@
+ARG ARCH=amd64
+ARG DISTRO=debian
+ARG DISTRO_VERSION=buster
+ARG VERSION=0
+
+FROM netdata/builder:${DISTRO}_${DISTRO_VERSION} AS build
```

We should probably remove the `_` here and update the Debian images to use that format (and a version number instead of a codename). Debian is really the only anomalous one here, all the others are already using `${DISTRO}${DISTRO_VERSION}` for the tag, and Debian is only the way it is right now because I was trying to match up with the tags we were using for the os-test repository.

---

I don't actually mind which way we do this. It will make no difference. Just bare in mind I'd like to get this whole flow working for "multiarch" too (_somehow_)

---

I'd rather have the numeric version number for the Debian images like we do for everything else. I'll update the tags on Docker Hub so that the new images get generated this way.

---

👍

---

Unresolving this as it's not been updated here.

Also, the image name is currently `netdata/package-builders` for the package builder images. `netdata/builder` is something different (I think it's used for the conainerized build tests). The tag names there are currently `${DISTRO_NAME}${DISTRO_VERSION}`, where the version is the standard numeric representation of the release version for the distro (8/910 for debian, 29/30/31 for fedora, 16.04/18.04/etc for Ubuntu, 15.0/15.1 for SuSE, etc).

---

Quick change to resolve the above conversation:

```suggestion
FROM netdata/package-builders:${DISTRO}${DISTRO_VERSION} AS build
```